aspnetcore/getting-started/index.md

```diff
@@ -47,7 +47,7 @@ dotnet dev-certs https --trust
 
 The preceding command displays the following dialog:
 
-![Security warning dialog](_static/cert.png)
+![Security warning dialog](~/getting-started/_static/cert.png)
 
 Select **Yes** if you agree to trust the development certificate.
```

Comment:
The formatting is off for the equivalent instruction in the macOS section -- the line "Password:*" above it is formatted the same as the following text, should be part of the preceding block.  Preexisting problem.